apps/backend/src/routes/home/+page.svelte

```diff
@@ -1,17 +1,23 @@
 <!-- JavaScript code -->
 <script lang="ts">
-  // import for fade feature
-  import { fade } from 'svelte/transition';
-
-  // import for FireStore Database Stuff
+  // Imports
+  import { doc, getDoc, setDoc, updateDoc, serverTimestamp } from 'firebase/firestore';
+  import type { FirebaseError } from 'firebase/app';
+  import type { FSUser, FSDomain } from '@armadillo/shared';
+  import { createForm } from 'svelte-forms-lib';
+  import { signOut } from '$lib/firebase/auth';
   import { firestore } from '$lib/firebase';
-  import { doc, getDoc } from 'firebase/firestore';
-  import { authStore } from '$lib/stores';
+  import { domainStore, authStore } from '$lib/stores';
+  import { fade } from 'svelte/transition';
+  import * as yup from 'yup';
 
   // State variables
   let showInfoCard = false;
   let selfDestructModal: HTMLDialogElement;
   let fileAccessModal: HTMLDialogElement;
+  let LogoutModal: HTMLDialogElement;
+  let SettingsModal: HTMLDialogElement;
```

Comment:
Only classes, interfaces, and types use TitleCase. Otherwise, please use camelCase

Reply:
Fixed, changed to
logoutModal and settingsModal